unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -84,6 +85,8 @@ export default class ActionBinder {
           await this.fillsign(files, eventName);
           break;
         case value.actionType === 'continueInApp':
+          this.LOADER_LIMIT = 100;
+          this.updateProgressBar(this.splashScreenEl, 100, 100);
```

Comment:
Looks good; except the unnecessary 3rd argument sent to the `this.updateProgressBar` method, as Narcis mentioned.
I will approve after that is fixed 👍